### DISCUSSION TITLE
Some Tailwind 2.0 questions...

### DISCUSSION THREAD

**Discussion by MartinHughes-BPC** [tailwindlabs/tailwindcss#2781] [General]
Hi All,

I have some Tailwind 2.0 questions:

1. I read somewhere (although I don't remember where) that TW 2.0 will drop support for Node 10 and 6(?). Does this only apply to what you guys are doing internally, or would I need to upgrade my end? The reason I ask is my server is an LTS SUSE Enterprise installation, which only supports Node 10 so I'll need to find a different publishing mechanism for my site.

2. Will the complex classes behaviour of @apply be the default in TW 2.0? If so, great - I find it very useful!
 
3. Will we still need the Tailwind/UI plugin or will this be integrated with TW 2.0?

Thanks,
Martin.

**Comment by hacknug** [comment]
> 3\. Will we still need the Tailwind/UI plugin or will this be integrated with TW 2.0?

RIP `@tailwindcss/ui`: https://twitter.com/adamwathan/status/1328487109435547648

**Comment by MartinHughes-BPC** [comment]
Just seen that TW 2.0 has dropped! So I can answer my own questions:

1) Yes I'll need to upgrade Node and change my deployment method.
2) Seems to work just fine without the "experimental" bit in the config.
3) Yes - but there are some other changes in Tailwind/UI worth noting in the TWUI docs.

**Reply by christopher-siegel** [reply]
Thanks for posting your own answer! Helps others out :)
